libs/server/src/lib/utils/read-projects.ts

```diff
@@ -74,14 +75,14 @@ export async function readBuilderSchema(
     path.join(npmPackage, 'package.json'),
     path.join(basedir, 'node_modules')
   );
-  const b = packageJson.json.builders;
+  const b = toLegacyFormat(packageJson.json).builders;
```

Comment:
toLegacyFormat only updates angular.json/workspace.json. You probably either have a similar one to package.json that renames generators into schematics and executors into builders or do it on the spot here.